executor/common_linux.h

```diff
@@ -1327,6 +1409,47 @@ static void netlink_wireguard_setup(void)
 	close(sock);
 }
 
+#if SYZ_EXECUTOR || SYZ_NIC_VF
+static void netlink_nicvf_setup(void)
+{
+	char cmdline[256];
+
+	if (!vf_intf.ppid)
+		return;
+
+	debug("ppid = %d, vf_intf.pass_thru_intf: %s\n",
+	      vf_intf.ppid, vf_intf.pass_thru_intf);
+
+	sprintf(cmdline, "nsenter -t 1 -n ip link set %s netns %d",
+		vf_intf.pass_thru_intf, getpid());
+	debug("%s\n", cmdline);
+	int ret = system(cmdline);
+
+	if (ret) {
+		debug("%s failed\n", cmdline);
+		return;
+	}
+	sprintf(cmdline, "ip a s %s", vf_intf.pass_thru_intf);
+	debug("%s\n", cmdline);
+	ret = system(cmdline);
+	if (ret) {
+		debug("%s failed\n", cmdline);
+		return;
+	}
+	sprintf(cmdline, "ip link set %s down", vf_intf.pass_thru_intf);
+	debug("%s\n", cmdline);
+	if (system(cmdline))
+		perror("system: ip link set down");
```

Comment:
We don't use `perror` anywhere else in executor. Wouldn't `debug` be also ok in this and similar cases?

Reply:
Requested change made - see: https://github.com/gkennedy12/syzkaller/tree/pull_072622_1